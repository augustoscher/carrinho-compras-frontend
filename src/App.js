import React from 'react';
import ActionBar from './components/ActionBar/ActionBar';
import CardButton from './components/ActionBar/CardButton';
import styled from 'styled-components';
import ProductList from './components/ProductList/ProductList';

const Content = styled.div`
  padding: 20px;
  display: flex;
  height: 100%;
`;

const ListContainer = styled.div`
  padding: 20px;
  width: 65%;
`;

const OrderContainer = styled.div`
  width: 35%;
  padding: 20px;
`;

function App() {
  return (
    <>
      <ActionBar>
        <CardButton text="Items" value="9" />
      </ActionBar>
      <Content>
        <ListContainer>
          <ProductList />
        </ListContainer>
        <OrderContainer>Cart</OrderContainer>
      </Content>
    </>
  );
}

export default App;
