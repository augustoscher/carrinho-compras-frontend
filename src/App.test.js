import React from 'react';
import { render } from '@testing-library/react';
import { MockedProvider } from '@apollo/client/testing';

import App from './App';

describe('App', () => {
  test('match snapshot', () => {
    const { asFragment } = render(
      <MockedProvider mocks={[]}>
        <App />
      </MockedProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
