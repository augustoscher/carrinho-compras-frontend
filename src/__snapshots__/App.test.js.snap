// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App match snapshot 1`] = `
<DocumentFragment>
  .c0 {
  background-color: #282c34;
  padding: 15px 15px;
  background-position: center;
  text-align: right;
  max-height: 50px;
}

.c1 {
  width: 100%;
}

.c2 {
  font-size: 1rem;
  font-weight: 600;
  text-transform: uppercase;
  padding: 12px 36px;
  cursor: pointer;
  background-color: #ffc107;
  border: 2px solid #ffc107;
  color: #4d4d4d;
}

.c2:hover:enabled {
  background-color: #c79100;
  border-color: #c79100;
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      <button
        class="c2"
      >
        Items: 9
      </button>
    </div>
  </div>
  .c0 {
  padding: 20px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
}

.c1 {
  padding: 20px;
  width: 65%;
}

.c2 {
  width: 35%;
  padding: 20px;
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      <p>
        Loading...
      </p>
    </div>
    <div
      class="c2"
    >
      Cart
    </div>
  </div>
</DocumentFragment>
`;
